scripts/utils/yarn.ts

```diff
@@ -20,7 +20,11 @@ export const addPackageResolutions = async ({ cwd, dryRun }: YarnOptions) => {
 
   const packageJsonPath = path.join(cwd, 'package.json');
   const packageJson = await readJSON(packageJsonPath);
-  packageJson.resolutions = { ...storybookVersions, 'enhanced-resolve': '~5.10.0' };
+  packageJson.resolutions = {
+    ...storybookVersions,
+    'enhanced-resolve': '~5.10.0', // TODO, remove this
+    playwright: '1.30.0', // this is for our CI test, ensure we use the same version as docker image, it should match version specified in `./code/package.json` and `.circleci/config.yml`
```

Comment:
Oh, maybe we can even import package json version and use it here? so we don't have to maintain two places and once one is updated, this is also updated?